assets/Config/languages.json

```diff
@@ -108,6 +108,22 @@
                             "عملیات"
                         ]
                     }
+                },
+                "competitors": {
+                    "header": {
+                        "button": "افزودن رقیب",
```

Comment:
The logic of this configuration is as follows: my entire program is an object that consists of a set of pages. Each page contains sections, and each section has separate components. Now, the text inside that component refers to what it is!

In the "competitors" page, the "header" section only has one button, so the key refers to being a button rather than indicating which button it is.